dist-git-client/dist_git_client.py

```diff
@@ -494,6 +570,8 @@ def main():
             srpm(args, config)
         elif args.action == "clone":
             clone(args, config)
+        elif args.action == "snapshot":
+            snapshot(args, config)
```

Comment:
Bikeshedding ... not sure if we should call this feature "snahpshot", or maybe "archive", or something else.